RNFBSDKExample/ios/RNFBSDKExample.xcodeproj/project.pbxproj

```diff
@@ -11,9 +11,9 @@
 		13B07FBC1A68108700A75B9A /* AppDelegate.m in Sources */ = {isa = PBXBuildFile; fileRef = 13B07FB01A68108700A75B9A /* AppDelegate.m */; };
 		13B07FBF1A68108700A75B9A /* Images.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 13B07FB51A68108700A75B9A /* Images.xcassets */; };
 		13B07FC11A68108700A75B9A /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = 13B07FB71A68108700A75B9A /* main.m */; };
-		6F42E5EA98685FBE727AD68D /* libPods-RNFBSDKExample-RNFBSDKExampleTests.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 5ABCB82D3C55B520AED9AD70 /* libPods-RNFBSDKExample-RNFBSDKExampleTests.a */; };
```

Comment:
Did you update project.pbxproj manually, or is this the result of https://github.com/thebergamo/react-native-fbsdk-next/blob/master/refresh-example.sh ?

Reply:
A result of the script